lib/modules/switch_board.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+import React from 'react-native';
+const { findNodeHandle } = React;
+const { ARSwitchBoardModule } = React.NativeModules;
+
+function presentNavigationViewController(component, route) {
```

Comment:
So Emission only provides the facility for React components to ask their host app to show a route, the actual routing and presenting of the VC is up to the host app.